packages/thirdweb/src/react/web/ui/TransactionButton/index.tsx

```diff
@@ -136,7 +136,7 @@ export function TransactionButton(props: TransactionButtonProps) {
           !unstyled
             ? {
                 opacity: !account || disabled ? 0.5 : 1,
-                minWidth: "150px",
```

Comment:
I don't think we can actually fix the width of the button. I can put any label I want for connect / transaction button. It's gonna get cutoff for sure. Not to mention our own localization.

Can we not fix the size jump while still having a flexible width? I think if you just set the min width to be a bit wider it would do what you're looking for